server/server.go

```diff
@@ -549,23 +556,23 @@ func (s *server) AddNode(ctx context.Context, req *rpcpb.AddNodeRequest) (*rpcpb
 	if req.StartRequest.LogLevel != nil {
 		logLevel = *req.StartRequest.LogLevel
 	}
-	if req.StartRequest.WhitelistedSubnets != nil {
-		whitelistedSubnets = *req.StartRequest.WhitelistedSubnets
-	}
-	if req.StartRequest.PluginDir != nil {
-		pluginDir = *req.StartRequest.PluginDir
-	}
+
+	// use same configs from other nodes
+	whitelistedSubnets = s.network.nodeInfos[s.network.nodeNames[0]].WhitelistedSubnets
```

Comment:
we should document this behavior

Reply:
You mean in README/docs right?
Or somewhere else

Reply:
and in the code, as necessary

Reply:
Updated README. Will also add to public docs.